### PR TITLE
fixes for noetic compile

### PR DESCRIPTION
**Pull request by davefeilseifer**

### CODE REVIEW

File: joint_trajectory_generator/CMakeLists.txt [hunk 1]
```diff
@@ -11,12 +11,13 @@ find_package(catkin REQUIRED COMPONENTS
...
 find_package(Boost REQUIRED COMPONENTS thread)
 find_package(orocos_kdl REQUIRED)
+#pkg_check_modules(OROCOS REQUIRED orocos_kdl)
 
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES joint_trajectory_generation
   CATKIN_DEPENDS roscpp urdf angles actionlib joint_trajectory_action pr2_controllers_msgs
-  DEPENDS Boost orocos_kdl)
+  DEPENDS Boost liborocos-kdl-dev)
```
Comment (v4hn) [inline joint_trajectory_generator/CMakeLists.txt:20]: That looks wrong to me. assuming `find_package(orocos_kdl)` above still declares `${orocos_kdl_INCLUDE_DIRS}` and `*_LIBRARIES` (as they are used below) you should not touch this line either. It simply records the values of these variables.

Comment (davefeilseifer) [reply joint_trajectory_generator/CMakeLists.txt:20]: I don't know why, but orocos_kdl no longer resolves in noetic. liborocos-kdl-dev seems to be the new dependency for the same needs as orocos_kdl provided

**Comment by v4hn** [comment]
> I don't know why, but orocos_kdl no longer resolves in noetic.
> liborocos-kdl-dev seems to be the new dependency for the same needs as orocos_kdl provided

That is because `orocos_kdl` in `CATKIN_DEPENDS` and in `package.xml` refers to a ROS package and `liborocos-kdl-dev` refers to the system installation.
In ROS melodic the module was packaged in ROS because the system-version was too old/broken.
noetic/Ubuntu uses the Ubuntu-version instead, so there is no ROS-specific package anymore.
However, `DEPENDS` - in contrast to `CATKIN_DEPENDS` does not know or care about ROS.
It simply collects the `${NAME}_LIBRARIES`/`${NAME}_INCLUDE_DIRS` variables and records them.
These variables are provided by the `find_package(orocos_kdl)` call above and are independent of whether a ROS-packaged or a system installation is used.

**Comment by davefeilseifer** [comment]
updated python and removed EOL distros, all checks are passing now.

**Comment by k-okada** [comment]
merged in #41

**Pull request closed without merging**